tests/examples/externals/definitions.py

```diff
@@ -0,0 +1,47 @@
+from dusk.script import *
+
+# A `definitions.py` could be one way to naturally introduce symbols that can
+# be imported and used in dusk stencils.
+
+regions = Regions(
+    # Here we specify which magic integers to use. This enables us to quickly
+    # prototype new features. Once SIR has built-in support for this, it can
+    # be changed.
+    lb=1,
+    nudging=2,
+    interior=3,
+    halo=4,
+)
+
+# string encoding
+another_regions = HorizontalRegions(
+    lb="lb", nudging="nudging", interior="interior", halo="halo"
+)
```

Comment:
Yea, I saw that redundancy too and line 21 resolves this. We'd still have to use strings though, since Python will complain about undefined names then.

```python
regions = Regions(lb, ..., halo) # raises NameError
```
```python
regions = Regions("lb", ..., "halo") # works and we can also create the right attributes to `regions`
```